zproject/config.py

```diff
@@ -46,11 +48,11 @@ def get_mandatory_secret(key: str) -> str:
 
 
 @overload
-def get_config(section: str, key: str, default_value: str) -> str: ...
+def get_config(section: str, key: str, default_value: str = "") -> str: ...
 @overload
-def get_config(section: str, key: str, default_value: Optional[str] = None) -> Optional[str]: ...
-def get_config(section: str, key: str, default_value: Optional[str] = None) -> Optional[str]:
-    return config_file.get(section, key, fallback=default_value)
+def get_config(section: str, key: str, default_value: bool) -> bool: ...
+def get_config(section: str, key: str, default_value: Union[str, bool] = "") -> Union[str, bool]:
+    return get_config_from_file(config_file, section, key, default_value)
```

Comment:
This has changed the default `default_value` value from `None` to `""`, which doesn’t seem like the documented intent.

Reply:
Thanks for the catch!  Added a commit to more explicitly align these two, which had different defaults.